packages/botonic-react/src/webchat/webchat.jsx

```diff
@@ -550,7 +550,7 @@ export const Webchat = forwardRef((props, ref) => {
         updateHandoff(handoff)
       }
       if (lastRoutePath) updateLastRoutePath(lastRoutePath)
-      updateLastMessageDate(new Date())
+      isOnline && updateLastMessageDate(new Date().toISOString())
```

Comment:
where is WebchatState.lastMessageUpdate? only by the bot?
Would it be possible to store the Date instead of the string? or would we have issues with browser state?

Reply:
storing Date may cause issues in localStorage, it's better to store a stringified iso version of the timestamp